templates/parameter.gotmpl

```diff
@@ -120,13 +120,13 @@ import (
   {{ end }}
 )
 
-// BusinessLogic{{ pascalize .Name }} executes the core logic of the related
+// Endpoint{{ pascalize .Name }} executes the core logic of the related
 // route endpoint.
-func BusinessLogic{{ pascalize .Name }}(f func(ctx *gin.Context{{ if .Params }}, params *{{ pascalize .Name }}Params{{ end }}) *api.Response) gin.HandlerFunc {
+func Endpoint{{ pascalize .Name }}(handler func(ctx *gin.Context{{ if .Params }}, params *{{ pascalize .Name }}Params{{ end }}) *api.Response) gin.HandlerFunc {
```

Comment:
nit: maybe it sounds better if `Endpoint` is in the end e.g. `CreateClusterEndpoint` rather than `EndpointCreateCluster`?

Reply:
Addressed, it makes sense.